test/HypLSP7.t.sol

```diff
@@ -28,7 +28,23 @@ import { PausableCircuitBreakerHook } from "../src/ISM/PausableCircuitBreakerHoo
 import { IERC725Y } from "@erc725/smart-contracts/contracts/interfaces/IERC725Y.sol";
 
 // constants
-import { _LSP4_METADATA_KEY } from "@lukso/lsp4-contracts/contracts/LSP4Constants.sol";
+import { _INTERFACEID_LSP0 } from "@lukso/lsp0-contracts/contracts/LSP0Constants.sol";
```

Comment:
Ah ok, since `@lukso/lsp0-contracts` is just used for testing, suggest putting this under the `devDependencies` then (so it is not shipped in the package build).

Reply:
after installing `@lukso/lsp0-contracts` for testing purposes, I couldn't build, due to missing lsp17 package, that's why  added it. Will move both of them to devDeps